src/common/components/cards/urls-card-row.tsx

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { NewTabLink } from 'common/components/new-tab-link';
```

Comment:
I think we'll need to use a `NewTabLinkWithConfirmationDialog` instead for the reports. Would probably be reasonable to have this file be agnostic with a `LinkComponent` dep.

Reply:
Now uses `LinkComponent` dep.